src/crypt/decrypter.ts

```diff
@@ -163,15 +163,23 @@ export default class Decrypter {
         return crypto.decrypt(data.buffer, aesKey);
       })
       .catch((err) => {
-        return this.onWebCryptoError(err, data, key, iv) as ArrayBuffer;
+        logger.warn(
+          '[decrypter.ts]: WebCrypto Error, disable WebCrypto API:',
+          err
+        );
+        return this.onWebCryptoError(data, key, iv) as ArrayBuffer;
       });
   }
 
-  private onWebCryptoError(err, data, key, iv): ArrayBuffer | null {
-    logger.warn('[decrypter.ts]: WebCrypto Error, disable WebCrypto API:', err);
+  private onWebCryptoError(data, key, iv): ArrayBuffer {
     this.config.enableSoftwareAES = true;
     this.logEnabled = true;
-    return this.softwareDecrypt(data, key, iv);
+    const result = this.softwareDecrypt(data, key, iv);
+    if (result === null) {
+      this.reset();
+      throw new Error("softwareDecrypt: result is 'null'");
```

Comment:
Double quotes should fail the linter check. You can use smart quotes here:

```suggestion
      throw new Error(`softwareDecrypt: result is 'null'`);
```